OneCommunityGlobal/HighestGoodNetworkApp
# fix make intangible hours editable

**BE PR** https://github.com/OneCommunityGlobal/HGNRest/pull/267
This PR is about making intangible hours editable.
User -> View Profile -> Volunteering Times
![BUG](https://user-images.githubusercontent.com/98206107/210458406-edeec260-5ee1-4d7a-b5af-f153d7c22424.JPG)


✔️ How to test this PR:
1. Login with an admin account, manually edit intangible hours and save changes.
2. Submit a new intangible time entry to see if intangible hours keep adding.
3. Change intangible entry to tangible entry and vice versa to see if intangible hours will increase/decrease accurately.

### UPDATED on 1/28:
1. Prevent users input negative numbers in hours field. (Include Weekly Committed Hours)
2. Improve the UI on the volunteering times tab. (Labels are not vertically centred)
![Before_UI_of_volunteering_times_tab](https://user-images.githubusercontent.com/98206107/215297595-3448d81d-16c8-43c7-9711-4e97aa597079.JPG)
![After_UI_of_volunteering_times_tab](https://user-images.githubusercontent.com/98206107/215297596-87c08f92-cf15-4aa9-bc3e-150fb51e4305.JPG)
